apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -66,6 +66,11 @@ export const UnconnectedTeacherHomepage = ({
    */
   const shouldShowAFEBanner = false;
 
+  /* We are hiding the Census banner to free up space on the Teacher Homepage (November 2023)
+   * when we want to show the Census banner again remove the next line
+   */
+  showCensusBanner = false;
```

Comment:
I'm a little surprised it lets you do this. To me, changing the value of a prop (sort of) feels really confusing. What do you think about using a different variable.

Naming is a bit tricky. I'd lean towards `forceHideCensusBanner = true` but that would require line 80 to change to `useState(showCensusBanner && !forceHideCensusBanner)`, which might be a little confusing. You could instead you a variable like `censusBannerAllowed = true` (not a big fan of this name, feel free to suggest others) then update line 80 to be `useState(showCensusBanner && censusBannerAllowed)`.

Thoughts?